projects/controllable_dialogue/mturk/run.py

```diff
@@ -118,7 +118,7 @@ def main():
     argparser.add_argument(
         '--mturk-log',
         type=str,
-        default=('$HOME/ParlAI/data/mturklogs/controllable/{}.log'.format(start_time)),
+        default=('data/mturklogs/controllable/{}.log'.format(start_time)),
```

Comment:
i'm not sure we should change this?

Reply:
Since commands are run from within parlai we want these logs to be created within the parlai project folder I assumed. For eg. For me ParlAI was not placed in HOME. So this created an unnecessary directory somewhere my project didn't exist. Also I conformed to the way this was done in wizards of wikipedia to keep things consistent, thought it was weird to have different implementations in different files (controllable vs wizard of wikipedia).

Could change it back if you would like, but I feel both files should be consistent with their location.

Reply:
ahh no don't worry about it, that change is fine then